contrib/examples/annotate-apply-time-mutations-inline-comment/Kptfile

```diff
@@ -0,0 +1,4 @@
+apiVersion: kpt.dev/v1
```

Comment:
This is an empty Kptfile, do we really need it?

Same comment for `annotate-apply-time-mutations-crd/Kptfile`

Reply:
Will the example tests run without it? I just copied what was already there.

Reply:
Tests pass without them.